openfe/protocols/openmm_utils/system_creation.py

```diff
@@ -132,7 +132,7 @@ def get_system_generator(
 
 def get_omm_modeller(protein_comp: Optional[ProteinComponent],
                      solvent_comp: Optional[SolventComponent],
-                     small_mols: list[SmallMoleculeComponent],
+                     small_mols: dict[SmallMoleculeComponent, OFFMol],
```

Comment:
@IAlibay ok it's a dict now.  This is the change to the non-Protocol code here, you're passing through pairs of SMC:OFF, the SMC for resid lookup and OFF which is carrying partial charges